webknossos-datastore/app/com/scalableminds/webknossos/datastore/datareaders/precomputed/PrecomputedCompressorFactory.scala

```diff
@@ -0,0 +1,16 @@
+package com.scalableminds.webknossos.datastore.datareaders.precomputed
+
+import com.scalableminds.webknossos.datastore.datareaders.{Compressor, JpegCompressor, NullCompressor}
+
+object PrecomputedCompressorFactory {
+  val nullCompressor = new NullCompressor
```

Comment:
Should be private. Also, are the compressors stateless? If so, the JpegCompressor could probably also be instantiated here already (possibly adapt this for zarr/n5 as well?).

Reply:
Does not seem to work.